### PR TITLE
[es] added names of municipios from Spain and Colombia

### PR DESCRIPTION
**Pull request by tatigf20**
List of multitoken _municipios_ from Spain and Colombia. @jaumeortola

### CODE REVIEW

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt [hunk 1]
```diff
@@ -1,4 +1,3754 @@
...
+Molina de Aragón;NPCSG00 #municipios de España
+Molina de Segura;NPCSG00 #municipios de España
+Molinos de Duero;NPCSG00 #municipios de España
+Molins de Rei;NPCSG00 #municipios de España
+Mollet de Peralada;NPCSG00 #municipios de España
+Mollet del Vallès;NPCSG00 #municipios de España
+Monasterio de la Sierra;NPCSG00 #municipios de España
+Monasterio de Rodilla;NPCSG00 #municipios de España
+Monasterio de Vega;NPCSG00 #municipios de España
+Monells i Sant Sadurní de l'Heura Cruïlles;NPCSG00 #municipios de España
```
Comment (jaumeortola) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt:1838]: -> Cruïlles, Monells i Sant Sadurní de l'Heura

https://ca.wikipedia.org/wiki/Cru%C3%AFlles,_Monells_i_Sant_Sadurn%C3%AD_de_l%27Heura

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt [hunk 2]
```diff
@@ -1,4 +1,3754 @@
...
+Navas del Madroño;NPCSG00 #municipios de España
+Navas del Rey;NPCSG00 #municipios de España
+Navia de Suarna;NPCSG00 #municipios de España
+Navianos de Valverde;NPCSG00 #municipios de España
+Negrilla de Palencia;NPCSG00 #municipios de España
+Negueira de Muñiz;NPCSG00 #municipios de España
+Neila de San Miguel;NPCSG00 #municipios de España
+Nieva de Cameros;NPCSG00 #municipios de España
+Noain (Elortzibar);NPCSG00 #municipios de España
+Noáin (Valle de Elorz);NPCSG00 #municipios de España
```
Comment (jaumeortola) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt:1971]: Parentheses are meaningless. Just accept both forms.

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt [hunk 3]
```diff
@@ -1,4 +1,3754 @@
...
+Pajares de los Oteros;NPCSG00 #municipios de España
+Palacios de Goda;NPCSG00 #municipios de España
+Palacios de la Sierra;NPCSG00 #municipios de España
+Palacios de la Valduerna;NPCSG00 #municipios de España
+Palacios de Riopisuerga;NPCSG00 #municipios de España
+Palacios de Sanabria;NPCSG00 #municipios de España
+Palacios del Arzobispo;NPCSG00 #municipios de España
+Palacios del Pan;NPCSG00 #municipios de España
+Palacios del Sil;NPCSG00 #municipios de España
+Palas de Rei;NPCSG00 #municipios de España
```
Comment (jaumeortola) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt:2063]: Placios -> MP

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt [hunk 4]
```diff
@@ -1,4 +1,3754 @@
...
+Puebla de San Miguel;NPCSG00 #municipios de España
+Puebla de Sanabria;NPCSG00 #municipios de España
+Puebla de Sancho Pérez;NPCSG00 #municipios de España
+Puebla de Valles;NPCSG00 #municipios de España
+Puebla de Yeltes;NPCSG00 #municipios de España
+Puebla del Maestre;NPCSG00 #municipios de España
+Puebla del Príncipe;NPCSG00 #municipios de España
+Puebla del Prior;NPCSG00 #municipios de España
+Puebla del Salvador;NPCSG00 #municipios de España
+Pueblica de Valverde;NPCSG00 #municipios de España
```
Comment (jaumeortola) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt:2259]: Puebla, Pueblica -> FS

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt [hunk 5]
```diff
@@ -1,4 +1,3754 @@
...
+Puerto de la Cruz;NPCSG00 #municipios de España
+Puerto de San Vicente;NPCSG00 #municipios de España
+Puerto de Santa Cruz;NPCSG00 #municipios de España
+Puerto del Rosario;NPCSG00 #municipios de España
+Puerto Lápice;NPCSG00 #municipios de España
+Puerto Lumbreras;NPCSG00 #municipios de España
+Puerto Moral;NPCSG00 #municipios de España
+Puerto Real;NPCSG00 #municipios de España
+Puerto Seguro;NPCSG00 #municipios de España
+Puerto Serrano;NPCSG00 #municipios de España
```
Comment (jaumeortola) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt:2281]: Puente, Puerto -> MS

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt [hunk 6]
```diff
@@ -1,4 +1,3754 @@
...
+Salas de los Infantes;NPCSG00 #municipios de España
+Salàs de Pallars;NPCSG00 #municipios de España
+Salceda de Caselas;NPCSG00 #municipios de España
+Saldaña de Burgos;NPCSG00 #municipios de España
+Sales de Llierca;NPCSG00 #municipios de España
+Salillas de Jalón;NPCSG00 #municipios de España
+Salinas de Oro;NPCSG00 #municipios de España
+Salinas de Pisuerga;NPCSG00 #municipios de España
+Salinas del Manzano;NPCSG00 #municipios de España
+Salinillas de Bureba;NPCSG00 #municipios de España
```
Comment (jaumeortola) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt:2421]: Salinas FP

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt [hunk 7]
```diff
@@ -1,4 +1,3754 @@
...
+Zizur Mayor;NPCSG00 #municipios de España
+Zizur Nagusia;NPCSG00 #municipios de España
+Zorita de la Frontera;NPCSG00 #municipios de España
+Zorita de los Canes;NPCSG00 #municipios de España
+Zorita del Maestrazgo;NPCSG00 #municipios de España
+Zotes del Páramo;NPCSG00 #municipios de España
+Agua de Dios;NPCSG00 #municipios de Colombia
+Agustín Codazzi;NPCSG00 #municipios de Colombia
+Alto Baudo;NPCSG00 #municipios de Colombia
+Altos del Rosario;NPCSG00 #municipios de Colombia
```
Comment (jaumeortola) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt:3478]: Altos MP

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt [hunk 8]
```diff
@@ -1,4 +1,3754 @@
...
+Puerto Santander;NPMSG00 #municipios de Colombia
+Puerto Tejada;NPMSG00 #municipios de Colombia
+Puerto Triunfo;NPMSG00 #municipios de Colombia
+Puerto Wilches;NPMSG00 #municipios de Colombia
+Rio Blanco;NPCSG00 #municipios de Colombia
+Río de Oro;NPCSG00 #municipios de Colombia
+Río Iro;NPCSG00 #municipios de Colombia
+Río Quito;NPCSG00 #municipios de Colombia
+Río Viejo;NPCSG00 #municipios de Colombia
+Roberto Payán;NPCSG00 #municipios de Colombia
```
Comment (jaumeortola) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt:3646]: Río MS

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt [hunk 9]
```diff
@@ -1,4 +1,3754 @@
...
+Santa Rosa del Sur;NPCSG00 #municipios de Colombia
+Santa Rosalía;NPCSG00 #municipios de Colombia
+Santa Sofía;NPCSG00 #municipios de Colombia
+Santafé de Antioquia;NPCSG00 #municipios de Colombia
+Santander de Quilichao;NPCSG00 #municipios de Colombia
+Santiago de Tolú;NPCSG00 #municipios de Colombia
+Santo Tomás;NPCSG00 #municipios de Colombia
+Talaigua Nuevo;NPCSG00 #municipios de Colombia
+Tolú Viejo;NPCSG00 #municipios de Colombia
+Unión Panamericana;NPCSG00 #municipios de Colombia
```
Comment (jaumeortola) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt:3740]: Unión FS

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt [hunk 10]
```diff
@@ -1,4 +1,3754 @@
...
+Valle de Guamez;NPCSG00 #municipios de Colombia
+Valle de San José;NPCSG00 #municipios de Colombia
+Valle de San Juan;NPCSG00 #municipios de Colombia
```
Comment (jaumeortola) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt:3743]: Valle MS

**Review by jaumeortola — APPROVED** [review]

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt (hunk 2 above)
Comment (tatigf20) [reply languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt:1971]: https://es.wikipedia.org/wiki/No%C3%A1in

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt [hunk 11]
```diff
@@ -3472,283 +3471,283 @@ Zorita de la Frontera;NPCSG00 #municipios de España
...
-Valle de San Juan;NPCSG00 #municipios de Colombia
-Vigía del Fuerte;NPCSG00 #municipios de Colombia
-Villa Caro;NPFSG00 #municipios de Colombia
-Villa de Leyva;NPFSG00 #municipios de Colombia
-Villa de San Diego de Ubate;NPFSG00 #municipios de Colombia
-Villa del Rosario;NPFSG00 #municipios de Colombia
-Villa Rica;NPFSG00 #municipios de Colombia
-Vista Hermosa;NPCSG00 #municipios de Colombia
-Zona Bananera;NPCSG00 #municipios de Colombia
+Agua de Dios;NPCSG00
```
Comment (jaumeortola) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt:3474]: the comment is removed?

Comment (tatigf20) [reply languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt:3474]: Oh, no!! I picked the wrong file 😢

Comment (tatigf20) [reply languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt]: It should be solved now. Thank you for the heads-up!

**Pull request merged**